### DISCUSSION TITLE
need a typical EBSD IPF color triangle

### DISCUSSION THREAD

**Discussion by mskyin2023** [mtex-toolbox/mtex#1784] [Ask Anything] [answered]
Hello everyone,

I am new here. i am trying MTEX to post-proceed my EBSD osc file. My material is bcc-ferrite steel. When i plot IPF map of EBSD scan and plot ipfKey and i got a different color shape as below. i used 432 as point group and specimen symmetry of 222. Did i make mistake? how can i get the traditional [001]-[110]-[111] color triangle? Thank you so much!
![image](https://user-images.githubusercontent.com/129394939/228835563-6c092b6f-398f-4a6b-a182-aed6ff243037.png)

QM

**Comment by lfgmorales** [comment]
Hi

mtexdata copper
ebsd('c').CS = ebsd('c').CS.Laue;
oM = ipfHSVKey(ebsd('c'))
oM.inversePoleFigureDirection = zvector;

The last two lines should do what you need


cheers


Luiz

**Comment by mskyin2023** [comment]
Thank you so much, Luiz!
I can plot as plot(oM) to get the traditional color triangle as below.
![image](https://user-images.githubusercontent.com/129394939/228876808-24ee6ebf-e486-473f-b7a8-b43ca09ee4a3.png)

i have questions: how to plot EBSD IPF map to use this tag of oM? and if i do not use this oM, as plot(ebsd, ebsd.orienations), the 2 maps look same? does this oM impact ODF calculation? sorry i have too many questions. Thank you again!

QM

**Comment by ralfHielscher** [comment, marked as answer]
You can do

```matlab
mtexdata copper
ebsd('c').CS = ebsd('c').CS.Laue;
oM = ipfHSVKey(ebsd('c'))
oM.inversePoleFigureDirection = zvector;
color = oM.orienation2color(ebsd.orientations);
plot(ebsd, color)
```
see also https://mtex-toolbox.github.io/EBSDSharpPlot.html
Ralf
```

**Reply by mskyin2023** [reply]
Hello Ralf,

Thanks for your instruction. i assume that parameter just influences EBSD IPF map, but does not influence ODF calculation? is this correct? What is default of ipfKey? HSV or other?   
Thank you and you both have a nice day!

QM

**Comment by lfgmorales** [comment]
No, the choice of colorcode does not influence your ODF,

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 30 Mar 2023, at 17:37, mskyin2023 ***@***.***> wrote:

﻿

Hello Ralf,

Thanks for your instruction. i assume that parameter just influences EBSD IPF map, but does not influence ODF calculation? is this correct? What is default of ipfKey? HSV or other?
Thank you and you both have a nice day!

QM

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1784#discussioncomment-5479242>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN46J2ATMV3IP4QYU33W6WSBPANCNFSM6AAAAAAWNHEK2Y>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1784#discussioncomment-5479242", "url": "https://github.com/mtex-toolbox/mtex/discussions/1784#discussioncomment-5479242", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by mskyin2023** [comment]
Great!! Thank you Luiz!!
